tensorflow_lite_support/python/task/audio/core/tensor_audio.py

```diff
@@ -95,26 +99,48 @@ def load_from_audio_record(self, record: audio_record.AudioRecord) -> None:
     data = record.read(self._buffer_size)
     self.load_from_array(data.astype(np.float32))
 
-  def load_from_array(self, src: np.ndarray) -> None:
-    """Loads audio data from a NumPy array.
+  def load_from_array(self,
+                      src: np.ndarray,
+                      offset: int = None,
+                      size: int = None) -> None:
+    """Loads the audio data from a NumPy array.
 
     Args:
-      src: A NumPy array contains the input audio.
-
+      src: A NumPy source array contains the input audio.
+      offset: An optional offset for loading a slice of the `src` array to the
+        buffer.
+      size: An optional size parameter denoting the number of samples to load
+        from the `src` array.
     Raises:
-      ValueError if the input audio is too large or if it contains an invalid
-      number of channels.
+      ValueError: Raised if the input array has an incorrect shape or if
+      offset + size exceeds the length of the `src` array.
```

Comment:
nit: put all parameters in a quote for API doc rendering later (e.g. \`offset\`)

Reply:
@khanhlvg Will do.